### PR TITLE
Battery level slider

### PR DESCRIPTION
**Pull request by g-ortuno**
Please review: @jyasskin @scheib 

battery level mock

### CODE REVIEW

File: app/src/main/res/drawable/battery_icon.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (scheib) [inline app/src/main/res/drawable/battery_icon.xml]: Where does the asset data come from? doc/attribute its source.

**Comment by scheib** [comment]
LGTM after asset data copyright is accounted for.

File: app/src/main/res/layout/activity_peripherals.xml [hunk 2]
```diff
@@ -20,20 +21,59 @@
...
+            android:layout_height="48dp"
+            android:layout_below="@id/textView_connectionStatus"/>
+    <LinearLayout
+            android:id="@+id/batteryLevel"
+            android:orientation="horizontal"
+            android:gravity="center_vertical"
+            android:layout_width="fill_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@id/textView_serviceName">
+        <ImageView
```
Comment (jyasskin) [inline app/src/main/res/layout/activity_peripherals.xml]: These can sit directly in the top-level `RelativeLayout`, using `layout_below="@id/batteryLevel"` and for the SeekBar `layout_toEndOf="@id/imageView_batteryIcon"`

**Comment by jyasskin** [comment]
LGTM

File: app/src/main/res/layout/activity_peripherals.xml (hunk 2 above)
Comment (g-ortuno) [inline app/src/main/res/layout/activity_peripherals.xml]: That LinearLayout only exist to vertically align the elements inside. I tried a couple of approaches with the upper level RelativeLayout but there seems to be no way to center-align the elements :disappointed:

Comment (jyasskin) [inline app/src/main/res/layout/activity_peripherals.xml]: Mm, yes. Thanks Android.

**Pull request merged**